styles.css

```diff
@@ -506,3 +541,113 @@ button.project-link-btn {
   font-family: 'Inter', Arial, Helvetica, sans-serif;
   line-height: 54px;
 }
+
+.contact-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2rem 5%;
+}
+
+.contact-comment {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  padding-bottom: 3rem;
+  color: #172b4d;
+  font-size: 2rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+}
```

Comment:
- [ ] Please make sure your contact form heading looks like that in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=F1n7sMh4ZV6XntRp-0) as per [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements): 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


your design:

<img width="367" alt="Screenshot 2023-03-28 at 9 03 39 PM" src="https://user-images.githubusercontent.com/44798044/228302752-819a1f79-9d0f-4ad7-a342-3dd228218d48.png">


figma design:


<img width="244" alt="Screenshot 2023-03-28 at 9 03 28 PM" src="https://user-images.githubusercontent.com/44798044/228302716-af46e49b-a86e-4622-b607-eb47f75d48b7.png">

Reply:
Hi, @shella12 

Kindly describe the problem in more detail, as to my knowledge, all the figma template design properties for this heading have been used in my project. 

![image](https://user-images.githubusercontent.com/117270680/228307614-a491d62d-c372-4e3b-a221-753954840fa6.png)

Reply:
Hi @sunga12 you are right you have used all the properties used in the figma design template but the task is to make your design look like the figma design as you can see [here](https://github.com/sunga12/My-Portfolio/pull/5#discussion_r1150864830), your heading text doesn't look like figma design. May be adding padding like `padding: 1rem;` or around might help.

Reply:
Alright, @shella12 are you referring to the font-family, or the spacing above/around the text in particular?

Reply:
@sunga12 It's the content's container itself, the one that has been highlighted in the inline comment. In your code that would be `.contact-comment`

Reply:
Alright, @shella12 I have added the margin-top for the .contact-comment, set to 3rem, is that what you were referring to?

Reply:
@sunga12 as you can see here the figma design text is aligned in the `center` of the container (referring to the sides, left and right)

<img width="435" alt="Screenshot 2023-03-28 at 10 11 49 PM" src="https://user-images.githubusercontent.com/44798044/228321978-de5449a8-a23b-4fb8-8e7b-6ad17ebb9f3f.png">


Your Design Still doesn't align properly in the center

<img width="366" alt="Screenshot 2023-03-28 at 10 32 43 PM" src="https://user-images.githubusercontent.com/44798044/228322735-b782bd55-1f70-4b70-b954-ba65b3ad3a2f.png">


I would suggest if you want to add margins please add margins on the left and right

Reply:
Hi, @shella12 , I understand what you mean now, however, the only way I can see to achieve that exact layout would be to set the .contact-comment width = 327px and height = 216px as seen in screenshot below (red border)


![image](https://user-images.githubusercontent.com/117270680/228324741-7e03a0ef-f2e3-4d59-bff7-01ddc22984e1.png)

So the size of the box stays the same regardless of the screen size of the mobile device. Is this how it is meant to be? Or should the box be able to respond to different screen widths?

Reply:
Well @sunga12 it's upto you how you want to make it, as long as it looks as its suppose to in different screen sizes, its okay you can use sizes like that but it's highly recommended that you use responsive values like %, rem, em etc